.github/workflows/rubocop.yml

```diff
@@ -41,3 +41,8 @@ jobs:
           num=`echo $files | wc -w`
           echo $files
           if [ $num -gt 0 ]; then bundle exec rubocop --config ./.rubocop.yml $files; else echo "No changed ruby files"; fi
+      - id: eslint
+        name: Run ESLint
+        run: |
+          yarn install --frozen-lockfile
```

Comment:
Doesn't the `Container Setup` already do this? If I understand correctly that runs `.github/workflows/container_setup/action.yml` which has a step `Install Node.js modules` which install the node modules

Reply:
It does, but for whatever reason it wasn't finding eslint in this context until I re-ran yarn install.
https://github.com/greenriver/hmis-warehouse/actions/runs/11914961227/job/33204111980

Reply:
hmm maybe a github caching issue